styles.css

```diff
@@ -1,3 +1,180 @@
+.header {
+  background-color: #aa8b56;
+  padding: 5px 10px 7px;
+  align-items: center;
+  position: fixed;
+  width: 100%;
+}
+
+.header .profile-pic {
+  width: 50px;
+  border-radius: 50%;
+  border-style: solid;
+  border-color: #2c3639;
+  margin-right: 20px;
+  display: flex;
+}
+
+.navbar {
+  width: 100%;
+  justify-content: flex-end;
+}
+
+.nav {
+  font-size: 20px;
+  color: #2c3639;
+  padding: 10px;
+}
+
+.nav:hover {
+  background-color: #395144;
+  color: #f0ebce;
+}
+
+#about-me {
+  background-image: url("image/nature-long.jpg");
+  background-size: 120%;
+  background-repeat: no-repeat;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  padding: 20% 10%;
```

Comment:
I'd be careful with percentage in padding. Very unintuitive as to what that means. Also, different screen sizes might use that very differently :)!